src/Umbraco.Tests.Common/Builders/PropertyBuilder.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using Umbraco.Core.Models;
+using Umbraco.Tests.Common.Builders.Interfaces;
+
+namespace Umbraco.Tests.Common.Builders
+{
+    public class PropertyBuilder
+        : BuilderBase<Property>,
```

Comment:
Any reason, this cannot be a `IProperty`

Reply:
No, seems to be fine with `IProperty`, have updated.